### ISSUE TITLE
Theming defective

### ISSUE THREAD

**Issue by arnonuem** [Antox/Antox#429]
When changing the theme within Antox (using settings dialog) the theme is only completely applied after a restart of the app.
Nevertheless the main problem is that the color of the settings dialog itself will not be changed. 
It stays purple.

**Comment by clazstudio** [comment]
For me _Navigation drawer_ also don't change color.

**Comment by wiiaam** [comment]
Can confirm, have had a go at fixing this but haven't been successful

**Comment by arnonuem** [comment]
Maybe an easy approach is changing the theme with a forced restart of the app. This would be OK for the first shot.

**Comment by wiiaam** [comment]
Yeah I almost had it almost fixed by making it dump the old activities and restart them after the theme was changed, although there were still parts that weren't changed. I feel there's something I'm overlooking for this, but I've also seen a few stack overflow issues on this as well with little resolve.
